HW/spark.py

```diff
@@ -0,0 +1,104 @@
+import sys
```

Comment:
So I couldn't get this running on my end because of an issue with my JDBC driver setup. I couldn't get my code to run either so it's totally possible that your code is fine and that it's just a problem with how I'm running it.